src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -46,8 +46,6 @@
                                   [:wallet/find-ens text contacts chain-id cb]
                                   300))
         :on-change-text        (fn [text]
-                                 (when-not (= scanned-address text)
```

Comment:
it's okay to remove this? 🤔 
cc @briansztamfater

Reply:
"scanned-address" and "text" will not be equal in case of multichain address